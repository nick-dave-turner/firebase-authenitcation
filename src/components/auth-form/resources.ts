interface Resources {
  title: string;
  subTitle: string;
  buttonLabel: string;
  footer: {
    text: string;
    link: string;
  };
}

export const SIGNIN_RESOURCES: Resources = {
  title: "Welcome back to FireBase Demo",
  subTitle: "Sign into your account to continue",
  buttonLabel: "Sign In",
  footer: {
    text: "Don't have an account? Sign Up",
    link: "/signup",
  },
};

export const SIGNUP_RESOURCES: Resources = {
  title: "Welcome to Firebase Demo",
  subTitle: "Create an account to start using Firebase Demo",
  buttonLabel: "Sign Up",
  footer: {
    text: "Already have an account? Sign In",
    link: "/signin",
  },
};
