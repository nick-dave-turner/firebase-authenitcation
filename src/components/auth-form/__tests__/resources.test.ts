import { SIGNIN_RESOURCES, SIGNUP_RESOURCES } from "../resources";

describe("AuthForm resources", () => {
  it("should snapshot SIGNIN_RESOURCES", () => {
    expect(SIGNIN_RESOURCES).toMatchSnapshot();
  });

  it("should snapshot SIGNUP_RESOURCES", () => {
    expect(SIGNUP_RESOURCES).toMatchSnapshot();
  });
});
