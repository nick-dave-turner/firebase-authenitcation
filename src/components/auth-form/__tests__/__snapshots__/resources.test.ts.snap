// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm resources should snapshot SIGNIN_RESOURCES 1`] = `
Object {
  "buttonLabel": "Sign In",
  "footer": Object {
    "link": "/signup",
    "text": "Don't have an account? Sign Up",
  },
  "subTitle": "Sign into your account to continue",
  "title": "Welcome back to FireBase Demo",
}
`;

exports[`AuthForm resources should snapshot SIGNUP_RESOURCES 1`] = `
Object {
  "buttonLabel": "Sign Up",
  "footer": Object {
    "link": "/signin",
    "text": "Already have an account? Sign In",
  },
  "subTitle": "Create an account to start using Firebase Demo",
  "title": "Welcome to Firebase Demo",
}
`;
